test/box_test.dart

```diff
@@ -52,6 +52,15 @@ void main() {
     expect(item.tString, equals('Two'));
   });
 
+  test('.put() cannot add duplicate values on a unique field', () {
+    final u1    = TestEntity.unique(uString: 'a', uLong: 1, uInt: 1,
+        uShort: 1, uBool: false, uByte: 1, uChar: 1);
+    final again = TestEntity.unique(uString: 'a', uLong: 1, uInt: 1,
+        uShort: 1, uBool: false, uByte: 1, uChar: 1);
+
+    expect(box.putMany([u1, again]), throwsException);
```

Comment:
Should check the exception message, no?

Reply:
That was more or less a mindfart placeholder. We can write proper tests now.

Reply:
I've fixed the test